server/re/src/reSysDataObjOpr.cpp

```diff
@@ -1321,10 +1323,7 @@ msiSetReServerNumProc(msParam_t* xnumProc, ruleExecInfo_t* rei)
         try {
             number_of_concurrent_executors = irods::get_advanced_setting<const int>(irods::CFG_NUMBER_OF_CONCURRENT_DELAY_RULE_EXECUTORS);
         }
-        catch (const irods::exception& e) {
-            irods::log(e);
-            return e.code();
-        }
+        catch (...) {}
```

Comment:
Should this microservice log that the config option isn't defined?

It makes sense for the delay server, but it's not clear whether a microservice should. This microservice will silently use the default.

I can see it going either way.

Reply:
let's log as WARNING - this will nudge administrators into the future and keep consistent with the best practices as the server continues to learn new tricks.

Reply:
done